src/app/components/albums-list/albums-list.component.html

```diff
@@ -3,8 +3,10 @@ <h1>Albums</h1>
     <section class="albums-list">
         <div *ngFor="let album of albums">
             <div class="details">
-                <h3>{{album.name}}</h3>
-                <img [src]="getCoverImage(album)" />
+                <a [routerLink]="['/album', album.id]" routerLinkActive="active">
```

Comment:
there is no need in `routerLinkActive="active"` as you will be transferred to another route